ui/console/src/main/scripts/plugins/metrics/plugins/metrics/ts/metricsPlugin.ts

```diff
@@ -69,6 +69,40 @@ module HawkularMetrics {
         };
     });
 
+    /**
+     * Replicates AngularJS 1.4 limitTo filter
```

Comment:
Right, but it doesn't have the "begin" argument which is handy for this situation. We can remove this if/when we upgrade to 1.4